### PR TITLE
Radio button fixes

### PR DESCRIPTION
**Pull request by Swiftyfast**
Fixed radio button problem, added address input options

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,13 +5,19 @@ import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 import axios from 'axios';
 
+var myObject = {"KIVA": "4"};
+
+
```
Comment (sandipsubedi) [inline src/index.js]: Is that space on purpose ?

File: src/index.js [hunk 2]
```diff
@@ -23,17 +29,21 @@ class FrontDoor extends React.Component{
...
     
       handleSubmit(event) {
+        var enteredAddress = this.state.value;
+        var sentAddress;
         var testURL = "http://dev-api.codeforkc.org//address-attributes/V0/1407%20Grand%20blvd?city=Kansas%20City&state=mo";
-        var dummyURL = "https://jsonplaceholder.typicode.com/users";
-        var newURL = "http://dev-api.codeforkc.org//address-attributes/V0/2516%20Holmes%20st?city=Kansas%20City&state=mo";
-        
-        console.log(this.state.value);
+       if (enteredAddress == ""){
```
Comment (sandipsubedi) [inline src/index.js]: it should be `===`

File: src/index.js [hunk 3]
```diff
@@ -142,5 +151,20 @@ class InsideHouse extends React.Component{
     }
 }
```
Comment (sandipsubedi) [inline src/index.js]: You should write a comment describing what this class does

File: src/index.js [hunk 4]
```diff
@@ -142,5 +151,20 @@ class InsideHouse extends React.Component{
...
+    constructor(props){
+        super(props);
+        this.state = {inputData: myObject.KIVA};
+    }
+
+    render(){
+        return(
+            <div>
+                <h1>Hey this part worked {this.state.inputData}</h1>
+                </div>
```
Comment (sandipsubedi) [inline src/index.js]: align this `div` with it's beginning so that they look better.

**Review by sandipsubedi — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 5]
```diff
@@ -71,64 +79,63 @@ class FrontDoor extends React.Component{
...
-            <label><input type="radio" name="myRadio" value="rad1" onChange={this.changeStuff} checked={this.state.doorway === 'rad1'} /><p1>Radio1</p1></label>
-            <label><input type="radio" name="myRadio" value="rad2" onChange={this.changeStuff} checked={this.state.doorway === 'rad2'}/><p1>Radio2</p1></label>
-            <label><input type="radio" name="myRadio" value="rad3" onChange={this.changeStuff} checked={this.state.doorway === 'rad3'}/><p1>Radio3</p1></label>
-        </div>
-        <h1>Here is the Second Radio Button</h1>
+        return(
+            <div>
+            <div className="menuBar">
+            <div><label><input type="radio" name="myRadio" value="rad1" onChange={this.changeStuff} checked={this.state.doorway === 'rad1'} /><p1>Kiva Pin (not working)</p1></label></div>
+                <div><label><input type="radio" name="myRadio" value="rad2" onChange={this.changeStuff} checked={this.state.doorway === 'rad2'}/><p1>Radio2</p1></label></div>
```
Comment (sandipsubedi) [inline src/index.js:116]: You should change `<p>` to `p`

**Review by sandipsubedi — CHANGES_REQUESTED** [review]

**Review by jzeilmann93 — APPROVED** [review]
Good. Change <p1> to <p>

**Review by sandipsubedi — APPROVED** [review]

**Pull request merged**